presto-product-tests/src/main/java/com/facebook/presto/tests/cli/PrestoCliUtils.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests.cli;
+
+import com.facebook.presto.cli.Presto;
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+import com.teradata.tempto.ProductTest;
+
+import java.io.IOException;
+import java.util.List;
+
+import static com.google.common.io.Resources.getResource;
+import static com.google.common.io.Resources.readLines;
+import static com.teradata.tempto.process.JavaProcessLauncher.defaultJavaProcessLauncher;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.Arrays.asList;
+
+public class PrestoCliUtils extends ProductTest
```

Comment:
This class doesn't actually have all the characteristics of a Utility class in Java (other classes need to extend this class). So rather rename this as something other than `PrestoCliUtils` such as `PrestoCliLauncher`.